src/common/noiseprofiles.c

```diff
@@ -56,7 +59,7 @@ JsonParser *dt_noiseprofile_init(const char *alternative)
   }
 
   // run over the file once to verify that it is sane
-  if(!dt_noiseprofile_verify(parser))
+  if(!_noiseprofile_verify(parser))
   {
     dt_control_log(_("noiseprofile file `%s' is not valid"), filename);
     dt_print(DT_DEBUG_ALWAYS, "[noiseprofile] error: `%s' is not a valid noiseprofile file. run with -d control for details", filename);
```

Comment:
Should the message now include '-d verbose' as well?

Reply:
Good catch :-)